recipes/libgpiod/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,6 @@
+#include "gpiod.hpp"
+
+int main(int argc, char *argv[]) {
+	gpiod::line l;
```

Comment:
```suggestion
	gpiod::line l;
```

Try using spaces instead of tab. Also, call a method to check the linkage.

Reply:
done